src/contexts/Balance/types.ts

```diff
@@ -0,0 +1,3 @@
+export interface ContextValues {
+  wpoktBalance?: any;
+}
```

Comment:
Actually four haha... Could we merge all of this into one file?

Reply:
Fixed in a0f4757d01ce1b4ff2b283fd8e81c6bacc09054f